Iryna_Ramanets/Module_1_Java/Java_Fundamentals/HW4/src/com/epam/calculator/test/TestRunner.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.calculator.test;
+
+import java.util.ArrayList;
+
+import org.testng.ITestNGListener;
+import org.testng.TestNG;
+import org.testng.xml.XmlSuite;
+
+public class TestRunner {
+
+	@SuppressWarnings("serial")
+	public static void main(String[] args) {
+		TestNG testNG = new TestNG();
+		testNG.addListener((ITestNGListener) new CustomTestListenerAdapter());
+
+		XmlSuite xmlSuitePositive = new XmlSuite();
+		xmlSuitePositive.setName("Calculator Tests suite parallel");
+		xmlSuitePositive.setSuiteFiles(new ArrayList<String>() {
```

Comment:
You forgot to add xml file with testng configuration for not parallel test running